Patch to unlock all channels with NO-IR mode for spectrum scan purposes.
Originally made by Kirill Lukonin (EvilWirelessMan)

Signed-off-by: Kirill Lukonin <klukonin@gmail.com>

--- a/db.txt
+++ b/db.txt
@@ -10,21 +10,11 @@ wmmrule ETSI:
 
 # This is the world regulatory domain
 country 00:
-	(2402 - 2472 @ 40), (20)
-	# Channel 12 - 13.
-	(2457 - 2482 @ 20), (20), NO-IR, AUTO-BW
-	# Channel 14. Only JP enables this and for 802.11b only
-	(2474 - 2494 @ 20), (20), NO-IR, NO-OFDM
-	# Channel 36 - 48
-	(5170 - 5250 @ 80), (20), AUTO-BW
-	# Channel 52 - 64
-	(5250 - 5330 @ 80), (20), NO-IR, DFS, AUTO-BW
-	# Channel 100 - 144
-	(5490 - 5730 @ 160), (20), NO-IR, DFS
-	# Channel 149 - 165
-	(5735 - 5835 @ 80), (20), NO-IR
-	# IEEE 802.11ad (60GHz), channels 1..3
-	(57240 - 63720 @ 2160), (0)
+	(2402 - 2494 @ 40), (20), NO-IR
+	# Channel 36 - 165
+	(5100 - 6000 @ 80), (20), NO-IR
+	# IEEE 802.11ad (60GHz), channels 1..4
+	(57000 - 71000 @ 2160), (0)
 
 
 country AD:
@@ -1281,17 +1271,12 @@ country UG: DFS-FCC
 	(5490 - 5730 @ 160), (24), DFS
 	(5735 - 5835 @ 80), (30)
 
-country US: DFS-FCC
-	(2402 - 2472 @ 40), (30)
-	# 5.15 ~ 5.25 GHz: 30 dBm for master mode, 23 dBm for clients
-	(5170 - 5250 @ 80), (23), AUTO-BW
-	(5250 - 5330 @ 80), (23), DFS, AUTO-BW
-	(5490 - 5730 @ 160), (23), DFS
-	(5735 - 5835 @ 80), (30)
-	# 60g band
-	# reference: section IV-D https://docs.fcc.gov/public/attachments/FCC-16-89A1.pdf
-	# channels 1-6 EIRP=40dBm(43dBm peak)
-	(57240 - 71000 @ 2160), (40)
+country US:
+	(2402 - 2494 @ 40), (20), NO-IR
+	# Channel 36 - 165
+	(5100 - 6000 @ 80), (20), NO-IR
+	# IEEE 802.11ad (60GHz), channels 1..4
+	(57000 - 71000 @ 2160), (0)
 
 country UY: DFS-FCC
 	(2402 - 2482 @ 40), (20)

